.github/workflows/ci.yml

```diff
@@ -105,12 +105,12 @@ jobs:
         uses: nrwl/nx-set-shas@v4
       - name: Install dependencies
         run: npm ci
-      - name: Set custom Chrome path for Windows only
-        if: matrix.os == 'windows-latest'
-        # This path is considered in `testing/setup/src/lib/chrome-path-setup.ts` and used in different test configurations
-        run: |
-          echo "CUSTOM_CHROME_PATH=C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe" | Out-File -FilePath $env:GITHUB_ENV -Append -Encoding utf8
-        shell: pwsh
+      #- name: Set custom Chrome path for Windows only
```

Comment:
Why not removing altogether?

Reply:
Thought about it! I could move it into the plugin docs under `CONTRIBUTORS.MD` to not loose the documentation as it took some time to figure it out.

Reply:
moved into docs